src/features/bookings/atoms/TicketCodeTitle.tsx

```diff
@@ -17,8 +17,8 @@ export const TicketCodeTitle = ({
   </TouchableOpacity>
 )
 
-const StyledTitle = styled(Typo.Title4).attrs(getHeadingAttrs(2))(({ theme }) => ({
+const StyledTitle = styled(Typo.Title2).attrs(getHeadingAttrs(2))(({ theme }) => ({
```

Comment:
Vu qu'on utilise un Title2 maintenant tu peux enlever `.attrs(...)`
```suggestion
const StyledTitle = styled(Typo.Title2)(({ theme }) => ({
```